g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationProductAction.java

```diff
@@ -211,14 +212,17 @@ public APIResponseDTO getPartialComTrans(String productName,
               throw new L3APIException(ConstantsMsg.TRANS_IS_NOT_FOUND);
           }else if(ja.size() == (reqLocaleSize*reqComponentSite)) {
               resulttranslationDTO.setBundles(ja);
-              return  super.handleResponse(APIResponseStatus.OK, resulttranslationDTO);
+              return super.handleVersionFallbackResponse(oldVersion, version, resulttranslationDTO);
           }else {
               for(JSONObject jsonNullObj:jsonNullList) {
                   ja.add(jsonNullObj);
               }
                resulttranslationDTO.setBundles(ja);
-              return  super.handleResponse(APIResponseStatus.MULTTRANSLATION_PART_CONTENT, resulttranslationDTO);
-              
+          	 if(oldVersion.equals(version)) {
+    			 return handleResponse(APIResponseStatus.MULTTRANSLATION_PART_CONTENT, resulttranslationDTO);
+    		 }else {
+    			 return handleResponse(APIResponseStatus.VERSION_FALLBACK_TRANSLATION, resulttranslationDTO);
+    		 }
```

Comment:
Please use handleVersionFallbackResponse

Reply:
this method return APIResponseStatus.MULTTRANSLATION_PART_CONTENT , not APIResponseStatus.OK. so can't use handleVersionFallbackResponse